gr-filter/lib/fft_filter.cc

```diff
@@ -210,6 +210,7 @@ void fft_filter_ccc::compute_sizes(int ntaps)
 
     if (VERBOSE) {
         gr::logger_ptr logger, debug_logger;
```

Comment:
I think this will work: you remove this line and the one you've just added, then change the `GR_LOG_ERROR` to use `d_logger` ... can you give that a go & see if it does indeed work? If so, let's use that here & anywhere else in this file .. correcting what Marcus did, since I think `d_logger` is available there too. Thx!

Reply:
Fixed and tested. I also changed the log from ERROR to ALERT, which seems more appropriate.